### PR TITLE
feat: Sign Presentation with P-256 DID and QR Fix

### LINKED ISSUE #117: Sign Presentation with P-256 DID

### PR DESCRIPTION
**Pull request by talwinder50**
1. Fix Chapi header and footer 
2. Generate presentation with ED25519 and P-256 DID Key 
3.  Display both the source and QR Code of presentation while generating vp. 

closes #117

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: cmd/user-agent/src/main.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ let defaultAriesStartupOpts = {
     assetsPath: '/aries-framework-go/assets',
     'outbound-transport': ['ws', 'http'],
     'transport-return-route': 'all',
-    'http-resolver-url': [],
+    'http-resolver-url': ["trustbloc@http://localhost:9080/1.0/identifiers"],
```
Comment (troyronda) [inline cmd/user-agent/src/main.js:47]: This looks like a local environment path.

Comment (fqutishat) [reply cmd/user-agent/src/main.js:47]: @talwinder50 remove value from here

Comment (fqutishat) [reply cmd/user-agent/src/main.js:47]: add it here https://github.com/trustbloc/edge-agent/blob/master/test/bdd/fixtures/agent-wasm/docker-compose.yml#L15

Comment (fqutishat) [reply cmd/user-agent/src/main.js:47]: https://github.com/trustbloc/edge-agent/blob/master/cmd/http-server/startcmd/start.go#L78

File: cmd/user-agent/src/main.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ let defaultAriesStartupOpts = {
     assetsPath: '/aries-framework-go/assets',
     'outbound-transport': ['ws', 'http'],
     'transport-return-route': 'all',
-    'http-resolver-url': [],
+    'http-resolver-url': [""],
```
Comment (fqutishat) [inline cmd/user-agent/src/main.js:47]: remove ""

Comment (talwinder50) [reply cmd/user-agent/src/main.js]: done

**Pull request merged**